src/components/ui/WithTooltip/WithTooltip.stories.mdx

```diff
@@ -0,0 +1,46 @@
+import { WithTooltip } from '.'
+
+<Meta title="UI Components/WithTooltip" />
+
+# WithTooltip
+
+Returns a higher order component (HOC) that renders a tooltip. Make sure to pass an existing element container ID and spread props of the wrapped component.
+
+The `WithTooltip` function takes a component to be wrapped, and an optional container ID used to insert the tooltip into.
```

Comment:
Should we explicitly call which components can get wrapped here, since not all support using this component right?

Reply:
I don't think that will be very maintainable, and will quickly get out of sync. The individual components should be repsonsible for that documentation IMO

Reply:
Okok, yeah makes sense, as long as individual components specify their support for WithToolTip we should be good.